app/assets/templates/header_tpl.jst.hbs

```diff
@@ -94,6 +94,7 @@
                 {{#if current_user.admin}}
                   <li><a href="/admins/dashboard">{{t "header.admin"}}</a></li>
                 {{/if}}
+                <li class="visible-sm visible-xs"><a href="/mobile/toggle">{{t "header.toggle_mobile"}}</a></li>
```

Comment:
Well there isn't already one for this text because it isn't used on any Handelbars template.